resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -147,13 +149,12 @@
       cuffnorm_output_table_headers.py ${replicates} ${sample_names}
 
       # Run normalizeRNASeq.
-      # TODO: Check what should be used for 'groupString' argument.
       Rscript --slave --no-save --no-restore-history -e " \
         library(RNASeqT); \
         normalizeRNASeq( \
           geneFPKMFile='genes.fpkm_table', \
           name='cuffnorm', \
-          groupString='${sample_names}', \
+          groupString='${labels}', \
```

Comment:
Has something changed and this will now work? Please elaborate.

Reply:
Is this something related to 7435626f3867d809d1e1f41a7399a5385c882ea3?

Reply:
If so, why didn't the tests fail? Please make sure that this is covered in tests.

Reply:
Yes, this was fixed in https://github.com/genialis/resolwe-bio/pull/352

Reply:
So it means that it is not covered in tests? Because otherwise, the tests should fail with the wrong behavior?

Reply:
The tool was changed but none of the Rscript outputs is covered in the tests.

Reply:
Does it make sense to include it in tests in some way?

Reply:
Maybe. I will expand the cuffnorm tests so they cover more